tests/test_fock.py

```diff
@@ -355,11 +355,54 @@ def SF_reference_trace(x, y):
         self.assertAlmostEqual(circuit(r1, r2), expected)
 
 
-if __name__ == '__main__':
-    # run the tests in this file
-    suite = unittest.TestSuite()
-    for t in (FockTests,):
-        ttt = unittest.TestLoader().loadTestsFromTestCase(t)
-        suite.addTests(ttt)
+class TestVariance:
+    """Test for the device variance"""
 
-    unittest.TextTestRunner().run(suite)
+    def test_first_order_cv(self):
+        """Test variance of a first order CV expectation value"""
+        dev = qml.device('strawberryfields.fock', wires=1, cutoff_dim=15)
+
+        @qml.qnode(dev)
+        def circuit(r, phi):
+            qml.Squeezing(r, 0, wires=0)
+            qml.Rotation(phi, wires=0)
+            return qml.var(qml.X(0))
+
+        r = 0.543
+        phi = -0.654
+
+        var = circuit(r, phi)
+        expected = np.exp(2*r)*np.sin(phi)**2 + np.exp(-2*r)*np.cos(phi)**2
+        assert np.allclose(var, expected, atol=1e-3, rtol=0)
```

Comment:
magic numbers?

Reply:
Hmm yeah I don't like this either. Just an issue with cutoff dimension - I could increase it beyond 15 to get better accuracy, but then the test takes longer. I'll try decreasing the mean photon number.

Reply:
yup that's easiest

Reply:
done